feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/BookmarkIcon.kt

```diff
@@ -54,10 +53,10 @@ fun BookmarkIcon(
         modifier = modifier,
         contentAlignment = Alignment.Center,
     ) {
-        if (progress in animationRange) {
+        if (state.isPlaying && !state.isAtEnd) {
```

Comment:
same as [L47](https://github.com/DroidKaigi/conference-app-2023/pull/1131/files#diff-5ab60002983aa4756e673f60b4537efd96e32a3ef541d84dd60778115ba16c28R47), we can judge whether animation is playing or not by `LottieAnimationState`